__tests__/pages/resource/id/evaluate.test.tsx

```diff
@@ -1,10 +1,49 @@
 import { render, screen, act } from "@testing-library/react";
 import "@testing-library/jest-dom";
-import { createMockRouter } from "../../../helpers/testHelpers";
+import { createMockRouter, getMockFetchImplementation } from "../../../helpers/testHelpers";
 import { RouterContext } from "next/dist/shared/lib/router-context";
 import EvaluateMeasurePage from "../../../../pages/[resourceType]/[id]/evaluate";
+import { fhirJson } from "@fhir-typescript/r4-core";
+
+const RESOURCE_ID_BODY: fhirJson.Bundle = {
+  resourceType: "Bundle",
+  meta: {
+    lastUpdated: "2022-06-23T19:52:58.721Z",
+  },
+  type: "searchset",
+  total: 2,
+  entry: [
```

Comment:
Here I go getting picky again, but can we change all the places is says diagnostic report here to Practitioner? Just makes things a bit less confusing and more accurate in the fixture.